index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Rominas Microverse Student Project 1</title>
+<link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+            <div class="logo">
+            <a href="#">My logo</a>
+            </div>
+        <img class="menu" src="images/Union.svg" alt="menu">
```

Comment:
- [OPTIONAL] I would suggest using a `nav` tag for your menu, as this will be used for navigation.